Documentation/reproducible-research.rst

```diff
@@ -131,3 +131,39 @@ SeisSol setups for community benchmark are described in the cookbook  (see :ref:
 
 .. include:: table_cookbook.rst
 
+
+We provide tutorial materials designed for software training, e.g.,  `CHEESE Advanced training <https://www.hlrs.de/training/2021-10-19-cheese/>`_ and `ICTP Advanced Workshop on Earthquake Fault Mechanics <https://indico.ictp.it/event/8715/overview>`_ .  The training material depends on an easy-to-obtain and uniform installation of SeisSol (Docker version) and related tools (e.g. Gmsh, Paraview) and necessary input files. 
```

Comment:
please change this to the following, while please including the link https://github.com/SeisSol/Training after 'SeisSol Docker container' 

"We provide the following small-scale examples, specifically designed for SeisSol training and tutorials, such as the  `ChEESE Advanced Training on HPC for Computational Seismology <https://www.hlrs.de/training/2021-10-19-cheese/>`_ and the `ICTP Advanced Workshop on Earthquake Fault Mechanics <https://indico.ictp.it/event/8715/overview>`_ .  These SeisSol training examples are part of the SeisSol Docker container (https://github.com/SeisSol/Training) which also includes related open-source tools (Gmsh, Paraview) and all required input files.